Fix formatting in helm values table (#7227)
circleci/circleci-docs: jekyll/_cci2/server/installation/installation-reference.adoc

|`insights_service.isEnabled`
|bool
|`true`
|Whether or not to enable the insights-service deployment.

|`insights_service.replicas`
|int
|`1`
|Number of replicas to deploy for the insights-service deployment.

|`internal_zone`
|string
|`"server.circleci.internal"`
|

|`keyset`
|object
|`{"encryption":"","signing":""}`
a| Keysets used to encrypt and sign artifacts generated by CircleCI. You need these values to configure server.

**Option 1:** Set the values `keyset.signing` and `keyset.encryption` here and CircleCI will create the Kubernetes Secret automatically. 

**Option 2:** Leave this blank, and create the secret yourself. CircleCI will assume it exists. The secret must be named 'signing-keys' and have the keys: `signing-key`, `encryption-key`.

|`keyset.encryption`
|string
|`""`
|Encryption Key. To generate an artifact ENCRYPTION key, run: `docker run circleci/server-keysets:latest generate encryption -a stdout`

|`keyset.signing`
|string
|`""`
|Signing Key. To generate an artifact SIGNING key, run: `docker run circleci/server-keysets:latest generate signing -a stdout`

|`kong.acme.email`
|string
|`"your-email@example.com"`
|

|`kong.acme.enabled`
|bool
|`false`

> |This setting will fetch and renew Let's Encrypt certs for you. It defaults to `false` as this only works when there's a valid DNS entry for your domain (and the app. sub domain) - so you will need to deploy with this turned off and set the DNS records first. You can then set this to true and run `helm upgrade` with the updated setting if you want.

Author: ```suggestion
|This setting will fetch and renew link:https://letsencrypt.org/[Let's Encrypt] certs for you. It defaults to `false` as this only works when there's a valid DNS entry for your domain (and the app. sub domain) - so you will need to deploy with this turned off and set the DNS records first. You can then set this to true and run `helm upgrade` with the updated setting if you want.
```

|`insights_service.isEnabled`
|bool
|`true`
|Whether or not to enable the insights-service deployment.

|`insights_service.replicas`
|int
|`1`
|Number of replicas to deploy for the insights-service deployment.

|`internal_zone`
|string
|`"server.circleci.internal"`
|

|`keyset`
|object
|`{"encryption":"","signing":""}`
a| Keysets used to encrypt and sign artifacts generated by CircleCI. You need these values to configure server.

**Option 1:** Set the values `keyset.signing` and `keyset.encryption` here and CircleCI will create the Kubernetes Secret automatically. 

**Option 2:** Leave this blank, and create the secret yourself. CircleCI will assume it exists. The secret must be named 'signing-keys' and have the keys: `signing-key`, `encryption-key`.

|`keyset.encryption`
|string
|`""`
|Encryption key to generate an artifact ENCRYPTION key run: `docker run circleci/server-keysets:latest generate encryption -a stdout`

|`keyset.signing`
|string
|`""`
|Signing Key To generate an artifact SIGNING key run: `docker run circleci/server-keysets:latest generate signing -a stdout`

|`kong.acme.email`
|string
|`"your-email@example.com"`
|

|`kong.acme.enabled`
|bool
|`false`

> |This setting will fetch and renew Let's Encrypt certs for you. It defaults to `false` as this only works when there's a valid DNS entry for your domain (and the app. sub domain) - so you will need to deploy with this turned off and set the DNS records first. You can then set this to true and run `helm upgrade` with the updated setting if you want.

Reviewer: Should we link out to Let's Encrypt here? Unless it's linked out to elsewhere on the page, I'd suggest it.

|object_storage.expireAfter
|int
|`0`
|number of days after which artifacts will expire.

|object_storage.gcs.service_account
|object
|`{"project_id":"... ...","type":"service_account"}`
a|GCP Storage (GCS) authentication config. 

**Option 1:** Set service_account with the service account JSON (raw JSON, not a string), and CircleCI will create the secret for you. 

**Option 2:** Leave the service_account field as its default, and create the secret yourself. CircleCI will assume it exists. 

**Option 3:** Leave the `service_account` field as its default, and set the workloadIdentity field with a service account email to use workload identities.

|object_storage.s3
|object
|`{"accessKey":"","enabled":false,"endpoint":"https://s3.us-east-1.amazonaws.com","irsaRole":"","secretKey":""}`
|S3 Configuration for Object Storage. Authentication methods: AWS Access/Secret Key, and IRSA Role.

|object_storage.s3.accessKey
|string
|`""`
a|AWS Authentication Config. 

**Option 1:** Set accessKey and secretKey here, and CircleCI will create the secret for you. 

**Option 2:** Leave accessKey and secretKey blank, and create the secret yourself. CircleCI will assume it exists. 

> **Option 3:** Leave accessKey and secretKey blank, and set the irsaRole field (IAM roles for service accounts), also set region: "your-aws-region".

Author: ```suggestion
**Option 3:** Leave accessKey and secretKey blank, set the irsaRole field (IAM roles for service accounts), and set region: `your-aws-region`.
```

----
{"bucketName":"","expireAfter":0,"gcs":{"enabled":false,"service_account":{"project_id":"... ...","type":"service_account"},"workloadIdentity":""},"s3":{"accessKey":"","enabled":false,"endpoint":"https://s3.us-east-1.amazonaws.com","irsaRole":"","secretKey":""}}
----

> |Object storage for build artifacts, audit logs, test results and more. One of 'object_storage.s3.enabled' or 'object_storage.gcs.enabled' must be true for the chart to function.

Reviewer: ```suggestion
|Object storage for build artifacts, audit logs, test results and more. One of `object_storage.s3.enabled` or `object_storage.gcs.enabled` must be true for the chart to function.
```

|object_storage.expireAfter
|int
|`0`
|number of days after which artifacts will expire.

|object_storage.gcs.service_account
|object
|`{"project_id":"... ...","type":"service_account"}`
a|GCP Storage (GCS) authentication config. 

**Option 1:** Set service_account with the service account JSON (raw JSON, not a string), and CircleCI will create the secret for you. 

**Option 2:** Leave the service_account field as its default, and create the secret yourself. CircleCI will assume it exists. 

**Option 3:** Leave the `service_account` field as its default, and set the workloadIdentity field with a service account email to use workload identities.

|object_storage.s3
|object
|`{"accessKey":"","enabled":false,"endpoint":"https://s3.us-east-1.amazonaws.com","irsaRole":"","secretKey":""}`
|S3 Configuration for Object Storage. Authentication methods: AWS Access/Secret Key, and IRSA Role.

|object_storage.s3.accessKey
|string
|`""`
a|AWS Authentication Config. 

**Option 1:** Set accessKey and secretKey here, and CircleCI will create the secret for you. 

**Option 2:** Leave accessKey and secretKey blank, and create the secret yourself. CircleCI will assume it exists. 

> **Option 3:** Leave accessKey and secretKey blank, and set the irsaRole field (IAM roles for service accounts), also set region: "your-aws-region".

Reviewer: Option 3 is a little confusing. Is the irsaRole directly related to the access and secret keys? The "...and set the..." makes it seem like the two are connected, but then it's ALSO asking to set the region. Can it just be

> Leave accessKey and secretKey blank, set the irsaRole field (IAM roles for service accounts), and set region: "your-aws-region".

Should it be "your-aws-region" or `your-aws-region`?

|vm_service.providers.ec2.accessKey
|string
|`""`
a|EC2 Authentication Config. 

**Option 1:** Set accessKey and secretKey here, and CircleCI will create the secret for you. 

**Option 2:** Leave accessKey and secretKey blank, and create the secret yourself. CircleCI will assume it exists. 

**Option 3:** Leave accessKey and secretKey blank, and set the irsaRole field (IAM roles for service accounts).

|vm_service.providers.ec2.enabled
|bool
|`false`
|Set to enable EC2 as a virtual machine provider.

|vm_service.providers.ec2.subnets
|list
|`["subnet-abcd1234"]`
|Subnets must be in the same availability zone.

|vm_service.providers.gcp.enabled
|bool
|`false`
|Set to enable GCP Compute as a VM provider.

|vm_service.providers.gcp.service_account
|object
|`{"project_id":"... ...","type":"service_account"}`
|GCP Compute Authentication Config. 

**Option 1:** Set service_account with the service account JSON (raw JSON, not a string), and CircleCI will create the secret for you. 

**Option 2:** Leave the service_account field as its default, and create the secret yourself. CircleCI will assume it exists. 

> **Option 3:** Leave the service_account field as its default, and set the workloadIdentityField with a service account email to use workload identities.

Reviewer: Should these be in backticks?

```suggestion
**Option 3:** Leave the `service_account` field as its default, and set the `workloadIdentityField` with a service account email to use workload identities.
```
